### PR TITLE
Feature/he 74 challenges info

### PR DESCRIPTION
**Pull request by Bluesmile82**
### Description
image

I had to use a state prop to only change one of the modals and a useEffect to remove the additional info when we change of tab.

### Designs
https://projects.invisionapp.com/d/main?origin=v7#/console/17484384/421888341/inspect?scrollOffset=4614
### Testing instructions
Check the modals. Open it with the '?' button next to the Challenges title
### Feature relevant tickets
https://half-earth-map.atlassian.net/browse/HE-74

### CODE REVIEW

**Comment by weberjavi** [comment]
@Bluesmile82 I'm taking that one :point_up: to address the changes on the metadata service agreed with Greta

**Comment by weberjavi** [comment]
@Bluesmile82 check the changes if you will :point_up:

File: src/components/modal-metadata/modal-metadata-component.jsx [hunk 1]
```diff
@@ -51,7 +51,9 @@ const ModalMetadata = ({ isOpen, handleClose, loading, title, metadata, addition
                   )
               }
             </dl>
-            {additionalContent}
+            {metadata && metadata.hasAdditionalContent === "TRUE" && (
```
Comment (Bluesmile82) [inline src/components/modal-metadata/modal-metadata-component.jsx:54]: Maybe we could parse this value before. But it's just an improvement

File: src/components/country-challenges-chart/country-challenges-modal-additional/country-challenges-modal-additional-component.jsx [hunk 2]
```diff
@@ -26,25 +26,6 @@ const CountryChallengesModalAdditional = () => {
       <div className={styles.countriesLegend}>
         {CONTINENTS.map((continent) => renderContinent(continent))}
       </div>
-      <div className={styles.sizesLegend}>
```
Comment (Bluesmile82) [inline src/components/country-challenges-chart/country-challenges-modal-additional/country-challenges-modal-additional-component.jsx]: There might be some styles code that we need to remove too

Comment (weberjavi) [reply src/components/country-challenges-chart/country-challenges-modal-additional/country-challenges-modal-additional-component.jsx]: :+1:

**Pull request merged**